components/browser/search/src/test/java/mozilla/components/browser/search/provider/AssetsSearchEngineProviderTest.kt

```diff
@@ -20,10 +21,8 @@ class AssetsSearchEngineProviderTest {
 
     @Test
     fun `Load search engines for en-US from assets`() = runBlocking {
-        val localizationProvider = object : SearchLocalizationProvider() {
-            override val country: String = "US"
-            override val language = "en"
-            override val region: String? = null
+        val localizationProvider = object : SearchLocalizationProvider {
+            override suspend fun determineRegion() = SearchLocalization("en", "US", "US")
```

Comment:
nit: As different provider repeat on the file, you may want to consider having them as properties like `usProvider` or `germanyProvider` :)

Reply:
Yeah, it's an option. I also like if test methods do not have overlapping dependencies and can be modified independently. Since this code existed before I'm keeping it for now. :)

Reply:
👍